### DISCUSSION TITLE
Researching API documentation tools

### DISCUSSION THREAD

**Discussion by roncodes** [fleetbase/fleetbase#113] [Ideas]
Found some interesting tools we could utilize to maintain the API documentation

1. https://apidog.com/
2. https://www.postman.com/
3. https://insomnia.rest/
4. https://hoppscotch.io/

The idea is we can build the API docs into a collection using one of these tools which could be exportable into many or different kinds of API documentation JSON/YAML formats which can be used to generate the API documentation on the api-docs repo. Or also share the collections as is.

**Comment by manishsainju** [comment]
Any though on using swagger?
